src/main/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformer.java

```diff
@@ -1026,4 +1044,8 @@ private void duplicateCaseCheck(String caseId, Map<String, Object> sscsCaseData,
             }
         }
     }
+
+    protected boolean notAValidFormType(String formType) {
+        return !stream(FormType.values()).anyMatch(formType1 -> formType1.getId().equalsIgnoreCase(formType));
```

Comment:
This needs to check all values except 'unknown'. It's not a valid value @bensonlievoco